app/controllers/subscriptions_controller.rb

```diff
@@ -54,7 +55,6 @@ def edit
   def create
     authorize :subscription
     @subscription = subscription_params
-    @subscription['EmailAddress'] = get_subscriber_email(@subscription['SubscriberId'])
```

Comment:
were you going to remove this from `update` also?

Reply:
Yeah that's part of line 84. But running some tests so I commented it out. But it will be back.

Reply:
its back.